pkg/controller/cincinnati/names.go

```diff
@@ -14,6 +14,10 @@ const (
 	NameInitContainerGraphData string = "graph-data"
 	// openshiftConfigNamespace is the name of openshift's configuration namespace
 	openshiftConfigNamespace = "openshift-config"
+	// NameDeploymentTrustedCA is the name of the copied ConfigMap from openshift-config
+	NameDeploymentTrustedCA = "trusted-ca"
```

Comment:
Calling this `NameDeployment...` is confusing given that it's always referring to the configmap.

```suggestion
	NameConfigMapTrustedCA = "trusted-ca"
```